### DISCUSSION TITLE
[Idea] Allow to increase the size of product images showed into order page (B.O.)

### DISCUSSION THREAD

**Discussion by darkf3d3** [PrestaShop/PrestaShop#31595] [Ideas]
### The problem

Product image showed into order page are very small, only 45x45 pixel.

### The solution

I think that would be usefull for a lot of merchant like me to have a setting that allow to change the size of these images.
Actually I set them to 180x180px, but it's a modify to the PS core file, so I have to do this on each upgrade.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
I like this idea! It's a common request from merchants.
